src/rust_lexer/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "rust_lexer"
+version = "0.1.0"
+authors = ["Aleksey Kladov <aleksey.kladov@gmail.com>"]
```

Comment:
You should copy-paste an existing file, e.g. [`src/libsyntax_pos/Cargo.toml`](https://github.com/rust-lang/rust/blob/master/src/libsyntax_pos/Cargo.toml).